src/components/networked.js

```diff
@@ -302,16 +323,89 @@ AFRAME.registerComponent('networked', {
               childEl.setAttribute(schema.component, schema.property, data);
             }
             else {
-              childEl.setAttribute(schema.component, data);
+              this.updateComponent(childEl, schema.component, data);
             }
           }
         } else {
-          el.setAttribute(key, data);
+          this.updateComponent(el, key, data);
         }
       }
     }
   },
 
+  updateComponent: function (el, key, data) {
+    switch(key) {
+      case "position":
+        var posComp = this.positionComponents.find((item) => item.el === el);
```

Comment:
I could see this find becoming problematic with ltos of components so we may want to index at some point but its probably fine for now.